### ISSUE TITLE
404 Error, No Import Button Shown...

### ISSUE THREAD

**Issue by PencilsDown** [ipfs/ipfs-webui#1918]
Under the files menu there is no option to import files...thank you for the help

![image](https://user-images.githubusercontent.com/102769859/161458793-afc857d3-e7c7-4630-90a5-a74904d0aeda.png)

 - OS: Windows 10
 - Browser : Chrome

**Smartphone (please complete the following information):**

**Additional context**

**Comment by lidel** [comment]
@PencilsDown  need more information on this one:


1. Are you able to run CLI tools against the same repo? if so, what is the output of `ipfs files stat /` ?

2. Are you able to open Developer tools via <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>i</kbd> and then go to Console and scroll down to the very last error (if any), and share screenshot here?
   It should look similar to this:
   > ![2022-04-05_01-19](https://user-images.githubusercontent.com/157609/161647947-d74a2477-a3a4-45e1-ac16-b48fdeb77896.png)
   Errors about locales can be ignored, you most likely will see additional one.

**Comment by PencilsDown** [comment]
@lidel appreciate the response...here is the error log screenshot, scrolled all the way to the bottom as stated....

![image](https://user-images.githubusercontent.com/102769859/161654022-f797d259-c1cf-4212-90a1-d1efffc0ef7d.png)

**Comment by bogosj** [comment]
I just ran into this problem with Windows Desktop 0.20.4. I installed 0.20.3 and it went away.

**Comment by PencilsDown** [comment]
> I just ran into this problem with Windows Desktop 0.20.4. I installed 0.20.3 and it went away.
> 
> 
> 
> Edit: This was a fresh install on a machine which had never had IPFS desktop installed.

@bogosj thank you sir, that was indeed the fix!

**Comment by lidel** [comment]
Interesting!

@PencilsDown @bogosj  are you using go-ipfs binary bundled with ipfs-desktop, or an external one?

**Comment by bogosj** [comment]
Bundled I assume. I did nothing other than install the .exe from the releases page.

**Comment by lidel** [comment]
Can you update to 0.20.5 and see if the problem is fixed there?

**Comment by bogosj** [comment]
I opened 0.20.3 before updating to 0.20.5 and encountered the original "null" error. On the status page it told me it couldn't connect to the API (unfortunately I forgot to screenshot) but I was also told to run `ipfs daemon`. I tried running it from PowerShell, Command Prompt, and another console emulator but I get:

```
'ipfs' is not recognized as an internal or external command,
operable program or batch file.
```

I installed 0.20.5 and it is working again. If the error resurfaces with this install I'll update.
